indexer-app/src/cmr/indexer/data/index_set.clj

```diff
@@ -209,6 +209,11 @@
    :equator-crossing-longitude m/double-field-mapping
    :equator-crossing-date-time m/date-field-mapping})
 
+(defnestedmapping track-pass-mapping
+  "Defines mappings for storing track pass."
+  {:pass m/int-field-mapping
```

Comment:
Any new fields for granules should use doc-values. I'm not sure how this works with nested fields though, should just the parent field use doc-values, or both the nested and parent?

Reply:
I think just within the nested object. Done.